src/protocolsupport/listeners/PlayerListener.java

```diff
@@ -76,10 +78,9 @@ public void onBlockPlace(BlockPlaceEvent event) {
 	@EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = true)
 	public void onEntityDamage(EntityDamageEvent event) {
 		if((event.getCause() == DamageCause.FIRE_TICK || event.getCause() == DamageCause.FIRE)) {
-			for(Player p : Bukkit.getOnlinePlayers()) {
-				//Send extra status packets to all old connections within 48 blocks (2304 = 48^2).
-				if(p.getLocation().distanceSquared(event.getEntity().getLocation()) <= 2304) { 
-					Connection connection = ProtocolSupportAPI.getConnection(p);
+			for(Entity e : event.getEntity().getWorld().getNearbyEntities(event.getEntity().getLocation(), 48, 128, 48)) {
```

Comment:
That's quite ineffective actually, the server itself has a method that allows getting nearby entities by class in a more effective way. Maybe that method should be added to PlatformUtils.